### ISSUE TITLE
[Bug][a11y] Highlighted What's new menu item not announced by TalkBack

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#7853]
## Steps to reproduce
1. Fresh install/update Fenix. Turn TalkBack on.
2. With TB on, tap the 3 dot menu on the homescreen.
3. Open the menu, tap the What's new option.
### Expected behavior
When What's new is highlighted (on new updates), TB announces the item as "highlighted". Same as it does on the 3 dot menu button.
### Actual behavior
TB announces the highlight on the 3 dot menu button, but not on the What's new option.

### Device information

* Android device:  Galaxy Note 8 (Android 9), Xiaomi Mi 8 Lite (Android 8.1)
* Fenix version: v3.2-beta1

**Comment by sblatz** [comment]
Highlight on the three dot menu should announce **why** it's highlighted. "Menu reader mode available." The item itself does not need to say "highlighted" 

If this preferred solution is not possible, please mark the inside menu as "highlighted"

**Comment by mcarare** [comment]
This is a duplicate of https://github.com/mozilla-mobile/fenix/issues/9795, marked as fixed.
